src/find_meaningful_words.py

```diff
@@ -0,0 +1,143 @@
+from collections import Counter
+from src.utils import SaveRead
+from src.count_words import get_words_only_small_letters, count_words_from_file
+
+#
+WORDS_PLACE_IN_FREQUENCY_LIST = 0
+FREQUENCY_PLACE_IN_FREQUENCY_LIST = 1
+
+# TODO: Add documentation to each function.
+# TODO: Add tests to each function
+# TODO: Divide function into different files.
+# TODO: Add configuration file for magic numbers and strings.
+
+
+def sort_words_count(counted_words_tuple_list):
```

Comment:
Changed to - sort_elements_and_occurences_tuple_list_from_high_to_low

I'm kinda thinking it's too long, maybe it would be hard to be more precise on what exactly this function is doing unless I would right something long?